vault/identity_store_oidc_provider_test.go

```diff
@@ -1090,6 +1302,30 @@ func TestOIDC_Path_OIDC_Authorize(t *testing.T) {
 				}(),
 			},
 		},
+		{
+			// This needs to be last because we do not reset authorization_code explicitly so it stays on the set false value after and messes with tests.
```

Comment:
I was confused by this.  I think you mean
```suggestion
			// This needs to be last because we do not reset authorization_code explicitly in testAuthorizeReq so
			//  it stays on the set false value after and messes with tests.
```

Reply:
I do not see the corresponding test for the default value of `client_credentials`.  Did I miss it?

Reply:
Authorize endpoint: default behavior is testes implicitly every call.

Token endpoint has tests in. `TestOIDC_Path_OIDC_Token_Client_Credentials_Flow` unless i am not understanding you correctly here.

Reply:
oh ok.

I can't find any angle how to test this.
At least I haven't found any other test that checks default Values. 

Since false is the default behavior missing json values I wouldn't test this.